pipelines/ApPipe.yaml

```diff
@@ -10,8 +10,13 @@ description: Stub Alert Production pipeline
 imports:
   - location: $AP_PIPE_DIR/pipelines/LSSTCam-imSim/ApPipe.yaml
 tasks:
-  rbClassify:
+  rewarpTemplate:
+    # TODO: remove in DM-50200 when goodSeeingCoadd is renamed to template_coadd
+    class: lsst.ip.diffim.getTemplate.GetTemplateTask
+    config:
+      connections.coaddExposures: goodSeeingCoadd
```

Comment:
Can you just use `parameters.coaddName` here (and therefore inherit the association from `ap_pipe`)?

Reply:
I could, but `ap_pipe` has moved entirely to `template_coadd` instead of any variant of `{coaddName}Coadd` so some override is necessary here. I wanted to be explicit in each CI package about what template each was currently using, with the expectation that would be updated on DM-50200. I worried that using `{coaddName}Coadd` would make it look more generic than it really is.